cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowActionNode.java

```diff
@@ -28,13 +29,21 @@
   private final WorkflowActionSpecification actionSpecification;
 
   public WorkflowActionNode(String nodeId, ScheduleProgramInfo program) {
```

Comment:
We have javadoc for the class. I added the javadoc to the constructors in the `WorkflowActionNode` as well. Is that what you meant? If so will add it to other node's constructors as well.